content/2019/BSDCan/mens-MQTT_for_system_administrators.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: slides
+title: "MQTT for system administrators (and for the IoT)"
+date: 2019-05-18
+author: Jan-Piet Mens
+email: jp@mens.de
+youtube: sHXKlBpIo_w
+venue: BSDCan 2019
```

Comment:
This one is also not a direct FreeBSD talk so it's better to check with the author first.

Reply:
i was on the fence before your comment.  i'll just remove it.   thanks!